pkg/agent/config/traffic_encap_mode.go

```diff
@@ -59,6 +59,9 @@ func GetTrafficEncapModes() []TrafficEncapModeType {
 
 // String returns value in string.
 func (m TrafficEncapModeType) String() string {
+	if m == TrafficEncapModeInvalid {
```

Comment:
Seems that this one is not necessary since we specify default value for it?

Reply:
I mentioned it in the commit message, `TrafficEncapModeInvalid` is not the issue we hit in ExternalNode, adding it because this object also has no string value, and it is expected to avoid similar errors in future logs since it is a supported type.